FK_solver/src/main.cpp

```diff
@@ -0,0 +1,23 @@
+#include "FK_solver.hpp"
+#include <deal.II/base/convergence_table.h>
+
+
+// Main function.
+int
+main(int argc, char *argv[])
+{
+    Utilities::MPI::MPI_InitFinalize mpi_init(argc, argv);
+   
+  const unsigned int degree = 2;
```

Comment:
Indentation here is a bit messy. While it's not a huge problem for this file, since it only has a few lines, you should always aim for well-indented code, because it massively improves readability and eases debugging, especially in projects larger than this.

I suggest you look into automatic indenting tools such as `clang-format` (which incidentally can be called from within most modern text editors - including VSCode - so that you automatically indent your code whenever you save the source file).